### PR TITLE
Add The Ultramarines Omnibus

### PR DESCRIPTION
**Pull request by Alex-Keyes**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -756,6 +756,13 @@ A short and rather old post-apocalyptic story which remained stuck in my mind li
 
 _Novels featuring the use use of technology, mainly weapons, for military purposes and principal characters that are members of a military organization involved in military activity; sometimes occurring in outer space or other planets._
 
+#### [The Ultramarines Omnibus](https://www.goodreads.com/book/show/31638.The_Ultramarines_Omnibus) (2008) *by [Graham Mcniell](https://en.wikipedia.org/wiki/Graham_McNeill)
```
Comment (RichardLitt) [inline readme.md]: Needs a closing asterix.

File: readme.md [hunk 2]
```diff
@@ -756,6 +756,13 @@ A short and rather old post-apocalyptic story which remained stuck in my mind li
...
+If you don't know anything about the warhammer universe, this book should provide a solid introduction.  It reals like Conan the Barbarian in space.  [@alex-keyes](https://github.com/alex-keyes)
```
Comment (RichardLitt) [inline readme.md]: Is it _awesome_ in itself, though? 

Also, "reals like"

Comment (Alex-Keyes) [reply readme.md]: I'd say the superhuman descendants of the ultimate warriors of mankind fighting lovecraftian horrors, alien forces seeking the desolation of the humans and various other neerdowells is pretty awesome.  I suppose I didn't make that clear enough.  40k isn't everyone's cup of tea, but the lore has an undeniable place in scifi.

Comment (RichardLitt) [reply readme.md]: I wouldn't know! I'm not familiar with Warhammer, besides knowing it exists. :)

**Comment by RichardLitt** [comment]
Added. Thanks, @Alex-Keyes \o/

**Pull request closed without merging**